### PR TITLE
Update View login and forgot password

### PR DESCRIPTION
**Pull request by sonpham2k**
## Related Tickets:
- [46260](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-intern-course-management/issues/46260)

## WHAT this PR does?
Tạo view login và view forgot password

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sonpham2k** [comment]
ready

File: resources/css/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    height: 320px;
+    border: 1px solid #9C9C9C;
+    background-color: #EAEAEA;
+  }
+  #login .container #login-row #login-column #login-box #login-form {
+    padding: 20px;
+  }
+  #login .container #login-row #login-column #login-box #login-form #register-link {
+    margin-top: -85px;
+  }
```
Comment (manhtd-1083) [inline resources/css/Login/login.css:19]:
```suggestion
  #login .container #login-row #login-column #login-box #login-form #register-link {
    margin-top: -85px;
  }
}

```
eof, indent 4 spaces

File: resources/views/Login/forgot.blade.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            </div>
+            <form class="card mt-4">
+                <div class="card-body">
+                    <div class="form-group"> <label for="email-for-pass">Enter your email address</label> <input class="form-control" type="text" id="email-for-pass" required=""> </div>
+                </div>
+                <div class="card-footer"> <button class="btn btn-success" type="submit">Get New Password</button> <a class="btn btn-danger" href="{{ route('login') }}">Back to Login</a> </div>
+            </form>
+        </div>
+    </div>
+</div>
```
Comment (manhtd-1083) [inline resources/views/Login/forgot.blade.php:18]: eof

File: resources/views/Login/forgot.blade.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="row justify-content-center">
+        <div class="col-lg-8 col-md-10">
+            <div class="forgot">
+                <h2>Forgot your password?</h2>
+            </div>
+            <form class="card mt-4">
+                <div class="card-body">
+                    <div class="form-group"> <label for="email-for-pass">Enter your email address</label> <input class="form-control" type="text" id="email-for-pass" required=""> </div>
+                </div>
+                <div class="card-footer"> <button class="btn btn-success" type="submit">Get New Password</button> <a class="btn btn-danger" href="{{ route('login') }}">Back to Login</a> </div>
```
Comment (manhtd-1083) [inline resources/views/Login/forgot.blade.php:14]: localization

File: resources/views/Login/head.blade.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Language
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+            <a class="dropdown-item" href="#"><img height="20px" width="20px" src="https://cdn-icons.flaticon.com/png/512/5373/premium/5373330.png?token=exp=1646021685~hmac=80350c5f5f2ddf177b24b693600b95fc"/>Tiếng việt</a>
+            <a class="dropdown-item" href="#"><img height="20px" width="20px" src="https://cdn-icons-png.flaticon.com/512/330/330425.png"/>English</a>
+            </div>
+        </li>
+    </ul>
+</nav>
```
Comment (manhtd-1083) [inline resources/views/Login/head.blade.php:13]: eof,

File: resources/views/Login/head.blade.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<nav class="navbar navbar-light" style="background-color: #e3f2fd;" >
+    <ul class="nav justify-content-end">
+        <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Language
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+            <a class="dropdown-item" href="#"><img height="20px" width="20px" src="https://cdn-icons.flaticon.com/png/512/5373/premium/5373330.png?token=exp=1646021685~hmac=80350c5f5f2ddf177b24b693600b95fc"/>Tiếng việt</a>
+            <a class="dropdown-item" href="#"><img height="20px" width="20px" src="https://cdn-icons-png.flaticon.com/512/330/330425.png"/>English</a>
```
Comment (manhtd-1083) [inline resources/views/Login/head.blade.php:9]: localization

File: resources/views/Login/layout.blade.php [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+    <link href="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+    <link href="stylesheet" type="text/css" href="../../css/Login/login.css"
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+</head>
+
+
```
Comment (manhtd-1083) [inline resources/views/Login/layout.blade.php:11]: xoá dòng thừa

File: resources/views/Login/layout.blade.php [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<head>
+    <link href="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+    <link href="stylesheet" type="text/css" href="../../css/Login/login.css"
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```
Comment (manhtd-1083) [inline resources/views/Login/layout.blade.php:8]: dùng npm, ko dùng link onl

File: resources/views/Login/login.blade.php [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+                <div id="login-column" class="col-md-6">
+                    <div id="login-box" class="col-md-12">
+                        <form id="login-form" class="form" action="" method="post">
+                            <h3 class="text-center text-info">Login</h3>
+                            <div class="form-group">
+                                <label for="username" class="text-info">Username:</label><br>
+                                <input type="text" name="username" id="username" class="form-control">
+                            </div>
+                            <div class="form-group">
+                                <label for="password" class="text-info">Password:</label><br>
```
Comment (manhtd-1083) [inline resources/views/Login/login.blade.php:18]: localization

File: resources/views/Login/login.blade.php [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+                            <div id="register-link" class="text-center">
+                                <a href="{{ route('forgot') }}" class="text-info">Forgot password</a>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
```
Comment (manhtd-1083) [inline resources/views/Login/login.blade.php:33]: eof

File: routes/web.php [hunk 10]
```diff
@@ -13,6 +13,10 @@
...
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('login', function () {
+    return view('Login/login');
+})->name('login');
+
+Route::get('forgot', function () {
+    return view('Login/forgot');
+})->name('forgot');
```
Comment (manhtd-1083) [inline routes/web.php:22]: tạo controller, ko viết trực tiếp vào route

**Pull request closed without merging**